Formula/redo.rb

```diff
@@ -11,15 +13,35 @@ class Redo < Formula
     sha256 "630ac52a05e6b4586f1f78219bc9bce9d17cd67ab1b4acccb3d62256c2839bce" => :sierra
   end
 
-  resource "docs" do
-    url "https://github.com/apenwarr/redo.git", :branch => "man"
+  depends_on "python@2"
+
+  resource "Markdown" do
+    url "https://files.pythonhosted.org/packages/51/3f/92f9d2f4a1d5da51e7808a469ab40c6cfdf3ba1013f56abb1f46677a655c/Markdown-3.1.tar.gz"
+    sha256 "fc4a6f69a656b8d858d7503bda633f4dd63c2d70cf80abdc6eafa64c4ae8c250"
+  end
+
+  resource "BeautifulSoup" do
+    url "https://files.pythonhosted.org/packages/1e/ee/295988deca1a5a7accd783d0dfe14524867e31abb05b6c0eeceee49c759d/BeautifulSoup-3.2.1.tar.gz"
+    sha256 "6a8cb4401111e011b579c8c52a51cdab970041cc543814bbd9577a4529fe1cdb"
+  end
+
+  patch do
```

Comment:
We can't have a patch without a fuller explanation, including when it should be removed.

Reply:
This PR has been merged, so presumably it will be in the next release... Still needs a comment explaining it and saying when it should be removed.

Reply:
It hasn't been merged yet. I'll add an explanation.

Reply:
Done in 6166b36.